mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/models/LegStep.java

```diff
@@ -325,4 +326,19 @@ public String getPronunciation() {
   public void setPronunciation(String pronunciation) {
     this.pronunciation = pronunciation;
   }
+
+  /**
+   * A String with the exit numbers or names of the way. Optionally included, if data is available
+   * @return String depicting exit
```

Comment:
check style will complain about there not being a line between the summary and `@return` tag.